src/renderer/components/_molecules/project-tree/index.tsx

```
@@ -253,30 +380,72 @@
+              className='focus:outline-none focus-visible:outline-none z-[100]'
+              onClick={(e) => e.stopPropagation()}
+            >
+              <div className='box flex h-fit w-[110px] flex-col text-xs rounded-lg bg-white dark:bg-neutral-950 text-neutral-1000 dark:text-neutral-300'>
+                {options.map((option) => (
+                  <div
+                    className='flex w-full cursor-pointer items-center gap-2 rounded-md px-2 py-1 hover:bg-neutral-100 dark:hover:bg-neutral-900'
+                    onClick={() => {
+                      option.onClick()
+                      setIsOpen(false)
+                    }}
+                  >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)